pkg/storageincentives/agent.go

```diff
@@ -415,6 +416,7 @@ func (a *Agent) handleSample(ctx context.Context, round uint64) (bool, error) {
 		return false, err
 	}
 
+	a.state.SetLastPlayedRound(round)
```

Comment:
I haven't chased the new code the whole way through, but what is the definition of "Played"?   With this Set here, it means that a sample was generated.  But to me, a node hasn't "Played" until it has successfully submitted the Commit transaction?   Or am I picking nits here?

Reply:
No you're not, that would be more accurate